x/subscription/keeper/cu_tracker.go

```diff
@@ -199,7 +199,7 @@ func (k Keeper) RewardAndResetCuTracker(ctx sdk.Context, cuTrackerTimerKeyBytes
 
 		// Note: if the reward function doesn't reward the provider
 		// because he was unstaked, we only print an error and not returning
-		providerReward, _, err := k.dualstakingKeeper.RewardProvidersAndDelegators(ctx, provider, chainID, sdk.NewCoins(creditToSub), types.ModuleName, false, false, false)
+		providerReward, _, err := k.dualstakingKeeper.RewardProvidersAndDelegators(ctx, provider, chainID, sdk.NewCoins(creditToSub), types.ModuleName, false, false, false, "subscription rewards")
```

Comment:
does it say from which subscription? for how much CU?

Reply:
done. see https://github.com/lavanet/lava/commit/93ab557e0b96a7d57b3bbf80dc59e744345bb9e5